test/integration/apiserver/certreload/certreload_test.go

```diff
@@ -184,12 +184,12 @@ func testClientCA(t *testing.T, recreate bool) {
 	defer tearDownFn()
 
 	// wait for request header info
-	err = wait.PollImmediate(100*time.Millisecond, 30*time.Second, waitForConfigMapCAContent(t, kubeClient, "requestheader-client-ca-file", "-----BEGIN CERTIFICATE-----", 1))
```

Comment:
I'm surprised these timeouts had to change... the change triggers here were file changes, which we're not touching, right?

Reply:
`waitForConfigMapCAContent()` appears to be fetching the cluster authn CM first to compare it with the expectation:
https://github.com/kubernetes/kubernetes/blob/4044f71121e5c51fa5cb12f6f98fe93c118f7745/test/integration/apiserver/certreload/certreload_test.go#L312-L329

I'll have a further look though, while I can't find it just now, it seems a bit odd we wouldn't wire the ClientCA and RequestHeaderCA dynamic certificate Notifier events to our queue.

Reply:
This was a proactive change

Reply:
OK, let's drop these changes to make this PR as small as possible and make sure that existing tests keep passing as is. I'm fine with a follow up to tweak these if we think that's necessary to make the test more robust.

Reply:
You're right, now that I think about it we did not touch anything with regards to how the CA files are handled, a change in behavior here would be weird.